src/main/java/core/basesyntax/service/impl/CsvParser.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.Parser;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvParser implements Parser {
+    private static final int FRUIT_POSITION = 1;
+    private static final int AMOUNT_POSITION = 2;
+    private static final int HEADER_LINES_LENGTH = 1;
```

Comment:
I think it is not length, better name like header position
If I understand it corectly

Reply:
I actually ment the length of header in lines